src/include/platform/CHIPDeviceConfig.h

```diff
@@ -693,7 +693,11 @@ static_assert(CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING_INTERVAL_MIN < CHIP_DEVICE_
  * Time in seconds that a factory new device will advertise commissionable node discovery.
  */
 #ifndef CHIP_DEVICE_CONFIG_DISCOVERY_TIMEOUT_SECS
+#if CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING == 0
```

Comment:
Can we please introduce the Zephyr's KConfig for this feature?

Reply:
option: we could probably have a Kconfig for advertising duration and turn on extended advertising if the duration > 15 minutes.

Reply:
I would also go with single config  (ext adv) or alternatively two configs (with timeout). The numbers 15 minutes and 48h are defined in the spec, so while configurability always is desired.. here it may result e.g. in failing the certification test case.

Reply:
The spec only mentions maximum duration of the announcement and that after 15 minutes it should switch to the extended one. Thus, a manufacturer can decide to advertise e.g. 1 hour if I read the spec correctly.

Reply:
Though, of course, we can go with a single boolean Kconfig if you think that will be easier. A customer can use chip_project_config.h to modify the duration, anyway.

Reply:
> The spec only mentions maximum duration of the announcement and that after 15 minutes it should switch to the extended one. Thus, a manufacturer can decide to advertise e.g. 1 hour if I read the spec correctly.

Okay, then it makes sense.